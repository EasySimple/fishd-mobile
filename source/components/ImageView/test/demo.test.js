import webDemoTest from '../../../tests/demoTest';

webDemoTest('ImageView');
