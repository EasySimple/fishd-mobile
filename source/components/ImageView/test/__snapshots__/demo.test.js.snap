// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders ImageView demo correctly 1`] = `
<div>
  <div
    class="components-tpl-demo-basic"
  >
    <div
      class="demo-title"
    >
      函数式用法
    </div>
    <div
      class="demo-card"
    >
      <a
        aria-disabled="false"
        class="fm-button fm-button-default fm-button-normal"
        role="button"
      >
        <span
          class="fm-button-text"
        >
          显示图片
        </span>
      </a>
    </div>
  </div>
  <div
    class="components-tpl-demo-basic"
  >
    <div
      class="demo-title"
    >
      组件式用法
    </div>
    <div
      class="demo-card"
    >
      <div
        class="demo-img__wrap"
      >
        <img
          alt="image"
          class="demo-img"
          src="https://placekitten.com/600/400"
        />
        <img
          alt="image"
          class="demo-img"
          src="https://placekitten.com/1200/900"
        />
      </div>
      <div
        class="fm-imageView"
      />
    </div>
  </div>
</div>
`;
